comment/models/comments.py

```diff
@@ -7,7 +7,7 @@
 
 
 class Comment(models.Model):
-    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, default=None)
+    user = models.ForeignKey(get_user_model(), on_delete=models.CASCADE, blank=True, null=True)
```

Comment:
We should add email field to the comment model.

Reply:
Any particular reason? As a matter of personal preference, I wanted to make these comments completely anonymous and hence didn't really want to save the email.

Reply:
Yes there is a good reason for that which is #43 
Email field is not only required for the anonymous comment but also for the authenticated user for the notification system later.

Reply:
I do see the feature request as a great one. Maybe we may put something like this somewhere that anonymous comment users won't be notified?

Reply:
why would you adopt that? when someone is commenting anonymously, still wanting to be notified for new updates in the thread. We can still implementing unsubscribe feature to remove them from emailing list.

Reply:
Yes I do get your point and it is a valid one. Okay maybe when we are implementing the notification thing, we may put a check where the user can choose to be notified. If they do choose to be notified, their email can be stored then.

I don't see a point of storing the email in the current state.

Reply:
Saving the comment without any reference (username or email) will give people the whole chance to abuse the use of comment and the flagging system will be in this case useless. And also any one can spam the thread with dirty stuff :D Therefore the comment must be saved with username or email.

Reply:
In my opinion, verifying by clicking on the link sent to their email makes spamming consistently more difficult. People can always spam a thread if they want to by creating an account.

If you see in essence, we are verifying their email account which is what mostly most websites do before account creating. Just the fact that we currently, do it before accepting every comment makes it more difficult to spam.

Reply:
For this reason we should implement blocking list in the future. For the coming features, I see the email field in the comment model is a must.